cclib/parser/molcasparser.py

```diff
@@ -441,6 +443,141 @@ def extract(self, inputfile, line):
+            self.skip_line(inputfile, 'stars')
+            line = next(inputfile)
+            
+            atomcoords = []
+            
+            line = next(inputfile)
+
+            while line.split() != []:
+                sline = line.split()
+                atomcoords.append(list(map(float, sline[1:])))
+                line = next(inputfile)
+
+            self.atomcoords.append(atomcoords)
+
+		#  **********************************************************************************************************************
+		#  *                                    Energy Statistics for Geometry Optimization                                     *
+		#  **********************************************************************************************************************
+		#                         Energy     Grad      Grad              Step                 Estimated   Geom       Hessian     
+		#  Iter      Energy       Change     Norm      Max    Element    Max     Element     Final Energy Update Update   Index  
+		#    1   -382.30023222  0.00000000 0.107221  0.039531 nrc047   0.085726  nrc047     -382.30533799 RS-RFO  None      0     
+		# ...
+		# ...		   
+		#   23   -382.30823115 -0.00000089 0.001030  0.000088 nrc053   0.000955  nrc053     -382.30823118 RS-RFO  BFGS      0    
+		#
+		#         +----------------------------------+----------------------------------+
+		#         +    Cartesian Displacements       +    Gradient in internals         +
+		#         +  Value      Threshold Converged? +  Value      Threshold Converged? +
+		#   +-----+----------------------------------+----------------------------------+
+		#   + RMS + 7.2395E-04  1.2000E-03     Yes   + 2.7516E-04  3.0000E-04     Yes   +
+		#   +-----+----------------------------------+----------------------------------+
+		#   + Max + 1.6918E-03  1.8000E-03     Yes   + 8.7768E-05  4.5000E-04     Yes   +
+		#   +-----+----------------------------------+----------------------------------+
+		#
+		#   Geometry is converged in  23 iterations to a Minimum Structure
+        if 'Geometry is converged' in line:
+            if not hasattr(self, 'optdone'):
+                self.optdone = []
+            self.optdone.append(len(self.atomcoords))
+
+		#   *********************************************************
+		#   * Nuclear coordinates of the final structure / Angstrom *
+		#   *********************************************************
+		#    ATOM              X               Y               Z     
+		#    C1               0.235547       -1.415838        0.012193
+		#    C2               1.313784       -0.488201        0.015297
+		#    C3               1.087036        0.895508        0.014333
+		# ...
+		# ...		
+		#    H19             -0.021315       -4.934913       -0.029666
+		#    H20             -1.431994       -3.721026       -0.041078
+        if 'Nuclear coordinates of the final structure / Angstrom' in line:
+            self.skip_line(inputfile, 'stars')
```

Comment:
These lines can be merged into one: `self.skip_lines(inputfile, ['b', 's'])`

Reply:
But it will give a message something like this:
```
[Molcas finalfiles/dvb_gopt.out WARNING] In molcasparser.py, line 508, line not blank as expected: ATOM              X               Y               Z
```
Is this alright?